app-laptop/msi-keyboard/msi-keyboard-1.0.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit udev
+
+DESCRIPTION="Control backlight of MSI laptop keyboards"
+HOMEPAGE="https://github.com/makkarpov/msi-keyboard"
+SRC_URI="https://github.com/makkarpov/msi-keyboard/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS=""
+
+RDEPEND="dev-libs/hidapi"
+
+src_install()
```

Comment:
This style doesn't follow the Gentoo defaults, you should stick to that for consistency.